### PR TITLE
New: (Indexer) Torrent lat-team.com

### PR DESCRIPTION
**Pull request by jaysonsantos**
#### Database Migration
NO

#### Description
This should add support to lat-team.com which is based on unit3d.
I'd like some guidance on where to put the tests and if I should dump JSON from the api to use as mocked results.

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

### CODE REVIEW

**Review by Qstick — APPROVED** [review]

File: src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+using NLog;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Definitions.UNIT3D;
+using NzbDrone.Core.Messaging.Events;
+
+namespace NzbDrone.Core.Indexers.Definitions
+{
+    public class LatTeam : Unit3dBase
+    {
+        public override string Name => "LatTeam";
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs:11]: Lat-Team

File: src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        public override string Language => "es";
+        public override string[] IndexerUrls => new[] { "https://lat-team.com/" };
+        public override string Description => "LatTeam is a Private Torrent Tracker for HD MOVIES / TV";
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs:14]: Lat-Team

**Review by Qstick — CHANGES_REQUESTED** [review]
Add dash on the two name instance, also needs a yml block in indexer definition update service.

File: src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Definitions.UNIT3D;
+using NzbDrone.Core.Messaging.Events;
+
+namespace NzbDrone.Core.Indexers.Definitions
+{
+    public class LatTeam : Unit3dBase
+    {
+        public override string Name => "LatTeam";
+        public override string Language => "es";
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs:12]: change to format as `{ISO 639-1}-{ISO 3166-1 alpha-2}`

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
> Add dash on the two name instance, also needs a yml block in indexer definition update service.

(This file & array)
https://github.com/Prowlarr/Prowlarr/blob/dbbc91380920c203514d536f5c4ace527ac391b4/src/NzbDrone.Core/IndexerVersions/IndexerDefinitionUpdateService.cs#L28

File: src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs (hunk 3 above)
Comment (jaysonsantos) [reply src/NzbDrone.Core/Indexers/Definitions/LatTeam.cs:12]: hey there, the thing is that idk in which country they are based in and there are like 20 ones that speak spanish in the americas, should i pick a random?

**Pull request merged**